waspc/waspls/test/Wasp/LSP/completionTests/route_to.golden

```diff
@@ -0,0 +1,4 @@
+Completion Items:
+  label={"todoApp"} kind={Just CiVariable} detail={Just ":: app (declaration type)"}
```

Comment:
This is how it actually works right now? This is a bit wrong, right -> because it offers any identifier in Wasp, even those that don't fit here? It is ok, just trying to remind myself.

Reply:
Yep! In this version it just lists all of the identifiers in the file that have a declaration type, which is something to improve in the future.